messages/pt.json

```diff
@@ -4,5 +4,7 @@
   "admin/editor.link.href.description": "URL para o link ex. \"https://www.google.com\", \"/sobre-nos\", etc",
   "admin/editor.product-link.title": "Link de produto",
   "admin/editor.link.target.title": "Onde abrir o link",
-  "admin/editor.link.target.description": "Você pode checar as opções disponíveis em: https://developer.mozilla.org/docs/Web/HTML/Element/a"
+  "admin/editor.link.target.description": "Você pode checar as opções disponíveis em: https://developer.mozilla.org/docs/Web/HTML/Element/a",
+  "admin/editor.link.rel.attribute.title": "\"rel\" atributo",
+  "admin/editor.link.rel.attribute.description": "Você pode checar as opções disponíveis em: https://www.w3schools.com/tags/att_a_rel.asp"
```

Comment:
Same here

Reply:
Sorry @victorhmp are you sure you don't mean at line 9 where there's the description of the rel attribute? The description you're highlighting was already there when I forked the repo

Reply:
Yep! I meant line 9 😅